src/main/java/com/kelinci/mainapp/api/UserController.java

```diff
@@ -15,55 +15,46 @@ public class UserController {
     private OurUser registeredUser;
```

Comment:
tutaj potrzebujemy dwóch ourUserow? W ogóle jak mamy listę ourUser'ow to chyba to już nie jest potrzebne. Co Ty na to?